node/include/cocaine/api/stream.hpp

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <cocaine/hpack/header.hpp>
```

Comment:
1. No, we don't.
2. IWYU. For example an implementation can provide its own methods, which encapsulate headers, and provides its API without requirement of headers definition.